src/twillio_utils.py

```diff
@@ -7,7 +7,8 @@
 from .data_utils import *
 from .responses import *
 from .news import return_news
-
+from .zip_code import get_zip_code_stats
```

Comment:
I would remove the `.` from here since it was giving some errors with the deploymebnt